w3lib/url.py

```diff
@@ -264,3 +264,8 @@ def any_to_uri(uri_or_path):
         return path_to_file_uri(uri_or_path)
     u = urlparse(uri_or_path)
     return uri_or_path if u.scheme else path_to_file_uri(uri_or_path)
+
+
+__all__ = ["add_or_replace_parameter", "any_to_uri", "file_uri_to_path",
```

Comment:
Exposing variables like `_safe_chars` could have made sense, but `import *` doesn't import them anyways, so no need to add them to `__all__`.

What's wrong with exposing `urljoin_rfc`? Not exposing it may break some code. The whole `__all__` fix is needed for backwards compatibility, so why fix it for some functions but not with others?